pyproject.toml

```diff
@@ -1,5 +1,92 @@
 [build-system]
-requires = ["cython", "numpy", "setuptools", "wheel"]
+requires = ["setuptools", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "pymt"
+description = "The CSDMS Python Modeling Toolkit"
+authors = [
+  {email = "mcflugen@gmail.com"},
+  {name = "The CSDMS team"}
+]
+maintainers = [
+  {email = "mcflugen@gmail.com"},
+  {name = "The CSDMS team"}
+]
+keywords = ["earth science", "model coupling"]
+license = {file = "LICENSE.rst"}
+classifiers = [
+  "Development Status :: 4 - Beta",
+  "Intended Audience :: Science/Research",
+  "License :: OSI Approved :: MIT License",
+  "Operating System :: OS Independent",
+  "Programming Language :: Cython",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: Implementation :: CPython",
```

Comment:
As a general comment, these Python versions tend to take a bit work to maintain, and generally don't have much benefit. Whereas `requires-python` (below) is actually checked when building the package, so has direct benefit. If there is _one_ useful classifier, it would be just `Programming Language :: Python :: 3 :: Only`.